Nikita_Konkov_Application/CSharpOOP3/CSharpOOP3/Cylinder.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP3
+{
+    internal class Cylinder: Shape
+    {
+        public string Name;
```

Comment:
Why are the fields public?

Reply:
It's not forbidden by home task. Why does it matter?